tests/algorithmBuild.js

```diff
@@ -4,195 +4,183 @@ const path = require('path');
-            const algName = pipelineRandomName(8).toLowerCase()
-            const pythonVersion = "python:3.5.10"
-
-            const buildStatusAlg = await buildAlgorithmAndWait({ code: code1, algName: algName, entry: entry, baseVersion: pythonVersion, algorithmArray: algList })
-            expect(buildStatusAlg.status).to.be.equal("completed")
-            const result = await runAlgGetResult(algName, [4])
-            await deleteAlgorithm(algName, true)
-            expect(result.data[0].result.sysVersion.toString()).to.be.equal("3,5,10,final,0")
-        }).timeout(1000 * 60 * 20)
-
-
-        it(`python 3.6`, async () => {
-            const entry = 'main36'
-            const algName = pipelineRandomName(8).toLowerCase()
-            const pythonVersion = "python:3.6"
-
-            const buildStatusAlg = await buildAlgorithmAndWait({ code: code1, algName: algName, entry: entry, baseVersion: pythonVersion, algorithmArray: algList })
-            expect(buildStatusAlg.status).to.be.equal("completed")
-            const result = await runAlgGetResult(algName, [4])
-            await deleteAlgorithm(algName, true)
-            expect(result.data[0].result.sysVersion.toString()).to.be.equal("3,6,15,final,0")
-        }).timeout(1000 * 60 * 20)
-
-        it(`python 3.7`, async () => {
-            const entry = 'main37'
-            const algName = pipelineRandomName(8).toLowerCase()
-            const pythonVersion = "python:3.7.16"
-
-            const buildStatusAlg = await buildAlgorithmAndWait({ code: code1, algName: algName, entry: entry, baseVersion: pythonVersion, algorithmArray: algList })
-            expect(buildStatusAlg.status).to.be.equal("completed")
-            const result = await runAlgGetResult(algName, [4])
-            await deleteAlgorithm(algName, true)
-            expect(result.data[0].result.sysVersion.toString()).to.be.equal("3,7,16,final,0")
-        }).timeout(1000 * 60 * 20)
-        it(`python 3.7-slim`, async () => {
-            const entry = 'main37slim'
-            const algName = pipelineRandomName(8).toLowerCase()
-            const pythonVersion = "python:3.7.16-slim"
-
-            const buildStatusAlg = await buildAlgorithmAndWait({ code: code1, algName: algName, entry: entry, baseVersion: pythonVersion, algorithmArray: algList })
-            expect(buildStatusAlg.status).to.be.equal("completed")
-            const result = await runAlgGetResult(algName, [4])
-            await deleteAlgorithm(algName, true)
-            expect(result.data[0].result.sysVersion.toString()).to.be.equal("3,7,16,final,0")
-        }).timeout(1000 * 60 * 20)
 
         const getBuildStates = async (jobId) => {
             const res = await chai.request(config.apiServerUrl)
                 .get(`/builds/status/${jobId}`);
-            return res.body.status
-
+            return res.body.status;
         }
+        // it(`python 3.5`, async () => { // NOT SUPPORTED, ONLY PYTHON >= 3.7
```

Comment:
3.5, 3.6 tests can be removed. No need to leave them as remarked

Reply:
Done